x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -132,7 +132,8 @@ export const SearchBar: FC<SearchBarProps> = (opts) => {
         setIsLoading(false);
 
         let aggregatedResults: GlobalSearchResult[] = [];
-        if (searchValue.length !== 0) {
+
+        if (searchValue.length !== 0 && searchValue.length <= 1000) {
```

Comment:
Super-nit: Should this be a config parameter (to be set in `kibana.yml`)?

`1000` seems like a high enough number to ever be hit, but I'm wondering if there's a user out there with a very complex query that they might need to bump this number at their own risk?

Of course, the potential workaround could be that they split the query using filters instead of writing the entire query in the search bar. But I'm trying to safe the next on-call person dealing with an SDH.

WDYT?

Reply:
Hey @afharo I'm taking this PR over whilst Rachel is out for PTO, Yea I think it does make sense to have this as a config parameter, and as such it been be made to be one.